extensions/amp-analytics/0.1/vendors/taboola.json

```diff
@@ -0,0 +1,19 @@
+{
+  "vars": {
+    "aid": ""
+  },
+  "requests": {
+    "url": "https://c3.taboola.com/amp/amp-analytics-taboola.html?url=${canonicalUrl}&aid=${aid}"
```

Comment:
I would suggest you do a more descriptive request name than, `url`

Reply:
Updated it to _loadamptrkhtml_ instead